values/values.antiaffinity.yaml

```diff
@@ -1,49 +1,96 @@
+cassandra:
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: app.kubernetes.io/component
+            operator: In
+            values:
+            - frontend
```

Comment:
this should maybe include all the temporal pods?

Reply:
Yes, but this functions without via transitive anti affinity for now